frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -107,7 +112,7 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
     return [
       combinedDisk.getName(),
       combinedDisk.getSourceValue(),
-      combinedDisk.getReadableSize(),
+      combinedDisk?.getSize() && humanizeBinaryBytes(combinedDisk?.getSize()?.value).string,
```

Comment:
```suggestion
      combinedDisk.getSize() && humanizeBinaryBytes(combinedDisk.getSize().value).string,
```

The `combinedDisk.getSize()` should always be defined.

In the right-hand side of `&&` logical operator, TS compiler already knows that expression `combinedDisk.getSize()` is not falsy, so you can do `.value` instead of `?.value`.